torchvision/prototype/transforms/functional/_utils.py

```diff
@@ -1,14 +1,17 @@
-from typing import Tuple, cast, Union
+from typing import Tuple, Union
 
 import PIL.Image
 import torch
 from torchvision.prototype import features
 from torchvision.transforms import functional as _F
 
+# FIXME
+
 
 def get_image_size(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> Tuple[int, int]:
     if type(image) is torch.Tensor or isinstance(image, PIL.Image.Image):
-        return cast(Tuple[int, int], tuple(_F.get_image_size(image)))
+        width, height = _F.get_image_size(image)
+        return height, width
```

Comment:
This is a major BC breaking change. Keeping the same method name and flipping the order is guaranteed to cause confusion.

My recommendation is to either undo OR offer a completely new method that returns the image_dims including channels in the order that you like.